indexer/variables.tf

```diff
@@ -362,3 +362,17 @@ variable "geoblocked_countries" {
   description = "Comma-delimited string of geoblocked countries"
   default     = "US,MM,CU,IR,KP,SY,CA"
 }
+
+variable "indexer_compliance_client" {
+  type        = string
+  description = "Client to use for compliance data, should be one of {BLOCKLIST | PLACEHOLDER | ELLIPTIC} defaults to ELLIPTIC"
```

Comment:
can you include where the blocklist is located? / what is placeholder?

Reply:
good catch on the blocklist not being wired in. Added block list variable and updated description.